source/main.cpp

```diff
@@ -32,17 +54,46 @@ int main(int argc, char** argv)
     static_cast<void>(argc);
     static_cast<void>(argv);
 
-    sh3_arc mft;
-    mft.Load();
+//    sh3_arc mft;
+//    mft.Load();
 
     Log(LogLevel::INFO, "===SILENT HILL 3 REDUX===");
     Log(LogLevel::INFO, "Copyright 2016-2017 Palm Studios\n");
 
     sh3_window window(640, 480, "sh3redux");
 
-    sh3_graphics::msbmp("data/pic/error.bmp");
+    bool quit = false;
+    SDL_Event e;
+
+    sh3_gl::program::load_error err;
+    sh3_gl::program test("test", err);
+    sh3_gl::vao<MyAttributes> myVao;
+
+    myVao.Bind();
+    sh3_gl::buffer_object verts = myVao[MyAttributes::VERTEX_SLOT];
```

Comment:
This can be a reference.
In fact I'm not sure if there's a good reason to have `buffer_object` and `vao` being copyable. We may just want to delete their copy ctor. If we find a use-case for it later, we can still change them to allow it.